src/transformers/pipelines.py

```diff
@@ -1175,6 +1176,121 @@ def span_to_answer(self, text: str, start: int, end: int):
+        max_length=140,
+        min_len=20,
+        no_repeat_ngram_size=3,
+        **generate_kwargs
+    ):
+        r"""
+        Args:
+            *documents: (list of strings) articles to be summarized
+            return_text: (bool, default=True) whether to add a decoded "summary_text" to each result
+            return_tensors: (bool, default=False) whether to return the raw "summary_token_ids" to each result
+
+            max_length: (`optional`) int
+                The max length of the sequence to be generated. Does not include tokens in input_ids.
+
+            num_beams: (`optional`) int
+                Number of beams for beam search. Must be between 1 and infinity. 1 means no beam search. Default to 1.
+            repetition_penalty: (`optional`) float
+                The parameter for repetition penalty. Between 1.0 and infinity. 1.0 means no penalty. Default to 1.0.
+            length_penalty: (`optional`) float Exponential penalty to the length. Default to 1.
+            num_return_sequences: (`optional`) int.
+                The number of independently computed returned sequences for each element in the batch. Default to 1.
+            min_len: (`optional`) int
+            no_repeat_ngram_size:  (`optional`) int. ban ngrams of this length from being repeated in the generated text
+            **generate_kwargs: extra kwargs passed to `self.model.generate`_
+
+        Returns:
+            list of dicts with 'summary_text' and/or 'summary_token_ids' for each document_to_summarize
+
+        .. _`self.model.generate`:
+            https://huggingface.co/transformers/model_doc/bart.html#transformers.BartForMaskedLM.generate
+
+        """
+        assert return_tensors or return_text
+        if self.framework == "tf":
+            raise NotImplementedError("Tensorflow not supported")
+        with self.device_placement():
+            inputs = self._parse_and_tokenize(*documents)
+            inputs = self.ensure_tensor_on_device(**inputs)
+            summaries = self.model.generate(
+                inputs["input_ids"],
+                attention_mask=inputs["attention_mask"],
+                num_beams=num_beams,
+                length_penalty=length_penalty,
+                max_length=max_length,
+                min_len=min_len,
+                no_repeat_ngram_size=no_repeat_ngram_size,
+                **generate_kwargs,
+            )
+            results = []
+            for summary in summaries:
+                record = {}
+                if return_tensors:
+                    record["summary_token_ids"] = summary
+                if return_text:
+                    record["summary_text"] = self.tokenizer.decode(
+                        summary, skip_special_tokens=True, clean_up_tokenization_spaces=False
+                    )
+                results.append(record)
+            return results
+
+    def _forward(self, *args, **kwargs):
+        raise NotImplementedError("Should not be called")
```

Comment:
The parent implements it, so I didn't want to inherit a method that isn't supported and have users expect it to work. 
I think since the method is private, I'll just delete this.